### PR TITLE
Jumpsuit for Paramedic

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
## About The Pull Request
Adds a Jumpsuit for the paramedic as no paramedic in the world runs around in surgery scrubs.

No, you already have seen the sprites in sprite pit.

## Why It's Good For The Game
Adds a nice green jumpsuit to better discern the paramedic from the rest of the staff. They arent a surgeon, they are outside of medbay usually and have to retrieve the people. Its only logical they have a sturdier jumpsuit.
In short:
Paramedic man now good.

## Changelog
:cl:
add: Sprite for new paramedic jumpsuit.
code: medsci, medical and the typical dmis
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/jobs/medsci.dm [hunk 1]
```diff
@@ -116,3 +116,18 @@
...
+	item_state = "paramedic"
+	permeability_coefficient = 0.50
+	armor = list(
+		melee = 10,
+		bullet = 5,
+		energy = 5,
+		bomb = 0,
+		bio = 30,
+		rad = 0
+	)
```
Comment (SpadesSlick) [inline code/modules/clothing/under/jobs/medsci.dm:133]: Please add newline.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 2]
```diff
@@ -116,3 +116,18 @@
...
 	icon_state = "psychturtle"
 	item_state = "b_suit"
+
+/obj/item/clothing/under/rank/paramedic
+	desc = "Jumpsuit of Moebius Medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
+	name = "moebius paramedic's jumpsuit"
+	icon_state = "paramedic"
+	item_state = "paramedic"
+	permeability_coefficient = 0.50
+	armor = list(
```
Comment (SpadesSlick) [inline code/modules/clothing/under/jobs/medsci.dm:126]: Armor is not added to uniform.

Comment (Smokingsquirrel) [reply code/modules/clothing/under/jobs/medsci.dm:126]: You know that every uniform for every department has armor values?

File: code/modules/clothing/under/jobs/medsci.dm (hunk 1 above)
Comment (Smokingsquirrel) [reply code/modules/clothing/under/jobs/medsci.dm:133]: what do you mean? Its the same format as all other uniforms?

Comment (Kurgis) [reply code/modules/clothing/under/jobs/medsci.dm:133]: We'll need to run some kind of git script to go through and add newlines to all files.

Comment (Smokingsquirrel) [reply code/modules/clothing/under/jobs/medsci.dm:133]: How does that even work? It's the first time i hear of this.

Comment (Kurgis) [reply code/modules/clothing/under/jobs/medsci.dm]: The comment was directed to Reere, apologies, should of specifically mentioned them.
